clients/gtest/blas2/her_gtest.yaml

```diff
@@ -27,7 +30,7 @@ Tests:
     alpha: *alpha_range
     matrix_size: *size_range
     incx: *incx_range
-    api: [ FORTRAN, C ]
+    api: [ FORTRAN, C, FORTRAN_64, C_64 ]
 
```

Comment:
We can argue fortran _64 suffices as it invokes C function only would miss double negation bug which would be very low probability.